lib/rets/client.rb

```diff
@@ -9,11 +9,13 @@ class Client
 
     def initialize(options)
       @options = options
-      @capabilities = nil
       clean_setup
     end
 
     def clean_setup
+      if options.fetch(:login_after_error, false)
```

Comment:
@dougcole good catch! I missed that. Yea, in order to function like it did before #136, the default should be `true`.